applications/portal/frontend/src/components/viewers/detailsViewer/DetailsViewer.tsx

```diff
@@ -0,0 +1,483 @@
+            fontWeight: 400
+        },
+        '&.MuiIconButton-root': {
+            padding: '0.5rem',
+            borderRadius: '0.375rem',
+            marginRight: '0.5rem'
+        }
+    },
+    textBlock: {
+        display: 'flex',
+        flexDirection: 'column',
+        fontSize: '0.75rem',
+        fontWeight: 400,
+        gap: '0.25rem'
+    },
+    populationColor: {
+        display: 'flex',
+        alignItems: 'center',
+        lineHeight: '0.938rem',
+        fontWeight: 400,
+        fontSize: '0.75rem',
+        paddingRight: '0.5rem',
+    },
+
+    square: {
+        width: '0.75rem',
+        height: '0.75rem',
+        borderRadius: '0.1rem',
+    },
+    pagination: {
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'space-between',
+        '& .MuiTypography-root': {
+            fontSize: '0.75rem',
+            fontWeight: 600,
+            lineHeight: '1rem',
+            color: populationSubTitleColor
+        },
+        '& .MuiPagination-ul li:not(:first-child, :last-child)': {
+            display: 'none'
+        },
+        padding: '0.75rem 1.5rem 1rem 1.5rem'
+    }
+});
+
+const DetailsViewer = (props: {
+    populationName: string | null,
+    populationColor: "#44C9C9"
+}) => {
+    const { populationName, populationColor } = props
+
+    const classes = useStyles();
+    const [tabIdx, setTabIdx] = React.useState(0);
+    const [anchorElMenu, setAnchorElMenu] = React.useState<null | HTMLElement>(null);
+    const [selectedIndex, setSelectedIndex] = React.useState(0);
+    const [category, setCategory] = React.useState('');
+    const [openDeleteDialog, setOpenDeleteDialog] = React.useState(false);
+    const [openUploadDialog, setOpenUploadDialog] = React.useState(false);
+    const [validationErrors, setValidationErrors] = React.useState(new Set([]));
+    let [page, setPage] = React.useState(1);
+    const [files, setFiles] = React.useState([])
+
+    const handleTabChange = (event: React.SyntheticEvent, newTabIdx: number) => {
+        setTabIdx(newTabIdx);
+    };
+
+    const getRGBAColor = () => {
+        return getRGBAFromHexAlpha(populationColor, 0)
+    }
+
+    const openMenu = Boolean(anchorElMenu);
+    const handleMenuClick = (event: React.MouseEvent<HTMLElement>) => {
+        setAnchorElMenu(event.currentTarget);
+    };
+    const handleMenuClose = () => {
+        setAnchorElMenu(null);
+    };
+    const handleMenuItemClick = (
+        event: React.MouseEvent<HTMLElement>,
+        index: number,
+    ) => {
+        setSelectedIndex(index);
+        setAnchorElMenu(null);
+    };
+
+    const handleCategoryChange = (e: any) => {
+        setCategory(e.target.value)
+    }
+
+    const handleAddFile = (type: string, value: any) => {
+        setFiles([...files, value])
+        validationErrors.delete(type)
+        setValidationErrors(validationErrors)
+    }
+
+    const handlePageChange = (event: any, page: number) => {
+        setPage(page)
```

Comment:
There's a linting rule that makes the build fail when we use shadow names (both page the parameter and page the state variable exist in this file).

Please run in the frontend folder:
```
 npx tslint -c tslint.json 'src/**/*.{ts,tsx}' --fix
```
to check and fix linting errors